### ISSUE TITLE
Shows success on checkout even though logs show failed on CCV2

### ISSUE THREAD

**Issue by Shooobz** [MrTpat/newegg-bot-public#2]
```
[Job: GSKILL Ram] New State: submitting_order
Submit order text: {"Result":"PlaceOrderFailure","MessageEntitys":[{"MessageCode":"Cvv2Invalid","Description":"Other issues","MessageDatas":null}]}
Submit order status: 200
[Job: GSKILL Ram] New State: submitted_order
[Job: GSKILL Ram] Finished
[Job: GSKILL Ram] Job GSKILL Ram succeeded!

```

Logs show above. Should be updated to notify on failure

**Comment by Shooobz** [comment]
Not sure how I could debug this (don't know much about web scraping), could it be that it's just not inserting the CCV2 for the credit card properly?

**Comment by Shooobz** [comment]
under newegg/communicator.py line 87:
`        data = {'SessionID':session_id,'Actions':[{'ActionType':'ConfirmPayment','JsonContent':json.dumps({'ActionType':'ConfirmPayment','Cvv2': cvv, 'TransactionNumber': transaction_number, 'PaytermsCode': 'Discover'})}],'EnableAsyncToken':True}`

Is Discover supposed to be hardcoded? cvv also translates to my city variable in the billing_config.json:
`(Pdb) print (cvv)
Bellerose
`

**Comment by Shooobz** [comment]
Bug on line 28 in profiles.py:

`cvv = d['city']`

**Comment by MrTpat** [comment]
Fixed in #3
